homeassistant/components/squeezebox/entity.py

```diff
@@ -0,0 +1,27 @@
+"""Base class for Squeezebox Sensor entities."""
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import EntityDescription
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .coordinator import LMSStatusDataUpdateCoordinator
+
+
+class StatusSensorEntity(CoordinatorEntity[LMSStatusDataUpdateCoordinator]):
+    """Defines a base staus sensor entity."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        device: DeviceInfo,
```

Comment:
Honestly, just create the device info in here instead of creating it in init and passing it to every object